### DISCUSSION TITLE
Does Crashlytics().crash() work on iOS simulator on debug build?

### DISCUSSION THREAD

**Discussion by naveendevs** [invertase/react-native-firebase#5318] [General]
Does Crashlytics().crash() work on iOS simulator on debug build?

**Comment by mikehardy** [comment]
You will love this blog I wrote about how to integrate (and test! most importantly) react-native-firebase crashlytics on all platforms.

https://invertase.io/blog/react-native-firebase-crashlytics-configuration

It definitely works - in a way you may easily verify - on iOS simulator in debug *if* you have it set up correctly.

**Reply by naveendevs** [reply]
Thanks, @mikehardy

It's a good explanation. I'm getting the release build reports as of now. But I would definitely set it up from scratch following your blog soon.
